MainDemo.Wpf/Buttons.xaml

```diff
@@ -111,7 +111,7 @@
                 <smtx:XamlDisplay Key="buttons_10">
                     <StackPanel Orientation="Horizontal">
                         <CheckBox Name="PopupEnabled">Enable Popup</CheckBox>
-                        <materialDesign:PopupBox Margin="16,0,16,0" StaysOpen="True" IsEnabled="{Binding IsChecked, ElementName=PopupEnabled}">
+                        <materialDesign:PopupBox Margin="16,0,16,0" StaysOpen="True" FlowDirection="LeftToRight" IsEnabled="{Binding IsChecked, ElementName=PopupEnabled}">
```

Comment:
FIX

Reply:
@SaimonSteisal i couldn't what you mean by FIX. Could you please be more specific?